tests/assets/function.yaml

```diff
@@ -7,11 +7,11 @@ inputs:
     - name: sky-file
       description: full path to output sky file
 operator: radiance-operator
-command: gensky -c -B '{{ inputs.parameters.desired-irradiance }}' > '{{ inputs.parameters.sky-file }}'
+command: gensky -c -B "{{ inputs.parameters.desired-irradiance }}" > "{{ inputs.parameters.sky-file }}"
```

Comment:
My understanding is that if `desired-irradiance=5` and `sky-file=file.sky` this will print as `gensky -c "5" > "file.sky"`. Is this the behavior we expect?

Reply:
My undrstanding is that it will be `gensky -c 5 > file.sky`. :)

Double quotes are required to make the variable being recognized from a dictionary. See here: https://github.com/argoproj/argo/tree/711bb11483a0ccb46600795c636c98d9c3a7f16c/examples#parameters

![image](https://user-images.githubusercontent.com/2915573/72667243-b54a5e80-39e7-11ea-9488-f2936f15cecd.png)

Reply:
I'm not sure that's the case if it's a template value within a string. ie: 'I really like {{something.i.like}}' doesn't need `"` because it's already within a string but `["{{something}}"] does because the whole template value will be a string.

Doesn't matter too much for now. Let's see how it pans out and worry about it when it breaks.